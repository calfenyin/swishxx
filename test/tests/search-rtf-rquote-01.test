search | | -i rtf.index | "here's" | 0
