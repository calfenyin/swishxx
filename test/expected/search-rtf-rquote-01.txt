# results: 1
100 ./sample.rtf 13394 This is a test RTF
